packages/common/src/hooks/useUSDCBalance.ts

```diff
@@ -48,15 +48,15 @@ export const useUSDCBalance = ({
   )
 
   const refresh = useCallback(async () => {
-    if (!ethAddress) {
+    if (!ethAddress || !sdk) {
```

Comment:
We talked about it offline, but strong preference for SDK having a consistent usage pattern (`(await sdk()).doSomething()`).